source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. An example would be a lightbulb, a Chromecast, or a smart hub.
+
+Home Assistant has a **discovery** feature, which scans for new devices as they’re added and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices like cell phones that can move around.
+
+## Entities
+
+Because many devices do more than one thing, most things in Home Assistant use  **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+Some entities are useful for controlling things in your home:
+
+- **Switches** let you check if something is switched on or off?
+- **Sensors** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **Binary sensors** detect something that is either on or off
+- **[Scripts](https://www.home-assistant.io/docs/scripts/)** are special entities that run custom code when they’re triggered
```

Comment:
Don't use full URL's instead pick them up from the root, for example: `/docs/scripts`.

It is unclear to me why scripts is linked, and others are not.

Reply:
Fair--this happened because of the markdown converter I used 😳

I also added more links, but there were lots of terms I couldn't find links for. If anyone can add links to the non-linked bold text I'd appreciate the help.